### PR TITLE
Mepts 653 apss c1 indicator

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Please fix the conflicts

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java [hunk 1]
```diff
@@ -446,4 +507,168 @@ private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
...
+    map.put(
+        "transferredFromOtherHealthFacilityWorkflowState",
+        hivMetadata
+            .getTransferredFromOtherHealthFacilityWorkflowState()
+            .getProgramWorkflowStateId());
+    map.put(
+        "adultoSeguimentoEncounterType",
+        hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId());
+    map.put(
+        "pediatriaSeguimentoEncounterType",
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:543]: @cfaife I cant find this in the specs

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java [hunk 2]
```diff
@@ -446,4 +507,168 @@ private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
...
+        hivMetadata.getPediatriaSeguimentoEncounterType().getEncounterTypeId());
+    map.put(
+        "arvPharmaciaEncounterType",
+        hivMetadata.getARVPharmaciaEncounterType().getEncounterTypeId());
+    map.put(
+        "masterCardEncounterType", hivMetadata.getMasterCardEncounterType().getEncounterTypeId());
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:549]: Cant find in the spec

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java [hunk 3]
```diff
@@ -446,4 +507,168 @@ private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
...
+            + "    AND e.voided = 0 "
+            + "    AND o.voided = 0 "
+            + "    AND e.encounter_type = ${prevencaoPositivaSeguimentoEncounterType} "
+            + "    AND (o.concept_id = ${memberShipPlanConcept} AND o.value_coded IN (${yesConcept},${noConcept}) "
+            + "        OR "
+            + "        o.concept_id = ${counseledOnSideEffectsOfArtConcept} AND o.value_coded IN (${yesConcept},${noConcept}) "
+            + "        OR "
+            + "        o.concept_id = ${adherenceEvaluationConcept} AND o.value_coded IN (${goodConcept}, ${arvAdherenceRiskConcept}, ${badConcept}) "
+            + "        ) "
+            + "    AND encounter_datetime "
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:577]: @cfaife this part of the query mas be the way you did with D1http://197.249.6.250:8080/fghmaster/module/htmlformentry/htmlForm.form?id=48

**Review by psmatsinhe — CHANGES_REQUESTED** [review]
Some changes to be done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java (hunk 1 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:543]: this is part of ARTStartDate in the B10 at Resumo Mensal, as it is  a calculation, I had to  extract the query in order to get the  `oldest date from the following dates`

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java (hunk 2 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:549]: this is part of ARTStartDate in the B10 at Resumo Mensal, as it is  a calculation, I had to  extract the query in order to get the  `oldest date from the following dates`

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java (hunk 3 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:577]: this C1 indicator

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java:577]: This works, as it asks for OR

**Review by psmatsinhe — APPROVED** [review]
Looks to me

**Pull request merged**